index.html

```diff
@@ -277,37 +277,83 @@ <h6 class="link-item">
-                    <li><a href="#" class="link-skill">Frameworks</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Frameworks</a></li>
+                </ul>
               </div>
-              <div class="skill-container">
-                <div class="skill-title">
-                  <ul>
-                    <li><a href="#" class="link-skill">Skills</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="vector-icon"></div>
+            </div>
+            <div class="skill-container">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Skills</a></li>
+                </ul>
               </div>
+              <div class="vector-icon"></div>
+            </div>
           </div>
-          </div>
+          <section id="contact-section">
+            <div class="content contact-container">
+              <div class="contact-title-container">
+                <h2 class="contact-title">Contact me</h2>
+              </div>
+              <div class="contact-description-container">
+                <p class="contact-description">
+                  If you have an application you are interedte in developing, a
+                  feature that you need built or a project that need coding. I'd
+                  love to help with it
+                </p>
+              </div>
+              <div class="form-container">
+                <form class="contact-form">
+<<<<<<< HEAD
+                  <input
+                    type="text"
+                    placeholder="Name"
+                    class="input-client-info"
+                  />
+                  <input
+                    type="email"
+                    name="email"
+                    placeholder="example@gmail.com"
+                    class="input-client-info"
+                  />
+                  <textarea
+                    name="message"
+                    id="message"
+                    placeholder="right your message here"
+                    class="client-message"
+                  >
+=======
+                  <input type="text" placeholder="Name" class="input-client-info" minlength="10" maxlength="30" required>
+                  <input type="email" name="email" placeholder="example@gmail.com" class="input-client-info" required>
+                  <textarea name="message" id="message" placeholder="right your message here" class="client-message" minlength="10" maxlength="500" required>
+>>>>>>> 8e928f3f1fac51db756f3d623fdf0ded38ce04f0
+                  </textarea>
+                  <div class="button-container">
+                    <button id="button" class="contact-button">
+                      <label for="button">Get It Touch</label>
```

Comment:
- [ ] To keep your HTML semantic, you shouldn't use a `<label>` tag inside a `<button>` tag. Keep in mind that if you have it like this, once you click the text, but not in the button itself, the submit won't be triggered. You can read more about labels in [here](https://html.spec.whatwg.org/multipage/forms.html).